app/services/topology_service.rb

```diff
@@ -36,4 +36,12 @@ def build_base_entity_data(entity)
      :kind   => entity_type(entity),
      :miq_id => entity.id}
   end
+
+  def populate_topology(topo_items, links, kinds)
+    topology = {}
+    topology[:items] = topo_items
+    topology[:relations] = links
+    topology[:kinds] = kinds
+    topology
```

Comment:
Minor, but this can be simplified to just

```ruby
def populate_topology(topo_items, links, kinds)
  {
    :items     => topo_items,
    :relations => links,
    :kinds     => kinds
  }
end
```

If you're going to do more in this area in the future then you can clean this up in a separate PR, or if not, perhaps here...it's up to you.